test/test_index_management.py

```diff
@@ -218,23 +218,12 @@ def test_case_5(self):
         coll0.drop_search_index("foo")
 
 
-if os.environ.get("TEST_INDEX_MANAGEMENT"):
```

Comment:
Can we remove all uses of TEST_INDEX_MANAGEMENT now?

Reply:
No, because we need the unified tests to run on a localhost, and the prose tests to run on atlas.

Reply:
I see, thanks.